jsk_arc2017_baxter/euslisp/lib/baxter.l

```diff
@@ -109,10 +109,11 @@
     (case (car args)
       ((:angle-vector)
        (let ((av (cadr args)) (joints (gethash arm hand-joints-)))
-         (when av
-           (dotimes (i (length av))
-             (send self (elt joints i) :joint-angle (elt av i))
-             (send self (elt joints (+ i (length av))) :joint-angle (elt av i))))
+         (if (and (null (eq (length av) 0)) (null (eq (length av) 2)))
+           (error "length of angle-vector must be 0 or 2.~%"))
```

Comment:
I think this should be error.
If ros::ros-error, motion continues and may cause fatal result such as finger collision.